src/state/routing/types.ts

```diff
@@ -21,6 +20,27 @@ export type PoolInRoute = {
   fee: string
   amountIn?: string
   amountOut?: string
+
+  // no used in the interface
+  address?: string
```

Comment:
Address is always returned by the api

Reply:
It is but the client side smart-order-router needs a provider to generate it and it's not actually used so I made it optional here.